standards/countries.rst

```diff
@@ -0,0 +1,44 @@
+.. _countries:
+
+A common list of countries
+==========================
+
+Having a standardized list of country names is an important prerequisite for region
+definitions, scenario analysis and model comparison.
+
+The :class:`nomenclature` package (`read the docs`_) builds on the :class:`pycountry`
```

Comment:
Since nomenclature and pycountry are not classes, sphinx recommends using :py:mod: (https://www.sphinx-doc.org/en/master/usage/domains/python.html#role-py-mod):
```suggestion
The :py:mod:`nomenclature` package (`read the docs`_) builds on the :py:mod:`pycountry`
```